controllers/components/modelregistry/modelregistry_support.go

```diff
@@ -0,0 +1,119 @@
+package modelregistry
+
+import (
+	"context"
+	"fmt"
+	"path"
+	"strings"
+	"text/template"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+
+	componentsv1 "github.com/opendatahub-io/opendatahub-operator/v2/apis/components/v1"
+	dsciv1 "github.com/opendatahub-io/opendatahub-operator/v2/apis/dscinitialization/v1"
+	infrav1 "github.com/opendatahub-io/opendatahub-operator/v2/apis/infrastructure/v1"
+	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/cluster"
+	odhtypes "github.com/opendatahub-io/opendatahub-operator/v2/pkg/controller/types"
+	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/conversion"
+	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/deploy"
+
+	_ "embed"
+)
+
+const (
+	ComponentName                   = componentsv1.ModelRegistryComponentName
+	DefaultModelRegistriesNamespace = "odh-model-registries"
```

Comment:
i must be blind, where is the create namespace logic now?

Reply:
https://github.com/opendatahub-io/opendatahub-operator/pull/1337/files#diff-7c54fcbca469fa317926436212d48e0a695fa8b7768e8c6822865594e0c0f21fR100-R105